includes/class-wp-smime.php

```diff
@@ -123,11 +135,16 @@ public static function encrypt ( $message, $headers, $certificates ) {
             $headers = explode( "\n", $headers );
         }
 
+        // Don't prepend content type since OpenSSL's function generates it's own headers
+	    $headers = array_filter( $headers, function ( $h ) {
```

Comment:
Please refactor so as not to use an anonymous function, which will break support for PHP <= 5.3. I realize it's end of life but WP Core still supports PHP 5.2 and I'd like to, too.